README.md

```diff
@@ -1 +1,59 @@
-# Conference-Page
\ No newline at end of file
+## Screenshot of the project
+
+<img src="/images/screenshot1.png">
+<img src="/images/screenshot2.png">
+
+## SawaSawa Fashion Week.
+
+- This is a website for SawaSawa Fashion Week that entails the program of the days it will be held.
+
+## Link to loom video
+
+- To view a viedo with a decsription of the project you can click on this [link](https://www.loom.com/share/a10a6043176d439bbc0b5a3ec0e202bb).
+
+## Built With
+
+- HTML & CSS & Javascript.
+
+### Prerequisites
+
+- Git. Which is a open source distributed version control system designed for source code management
+- A text editor (e.g Visual Studio Code, Vim, Atom & Sublime.)
+- A web browser (e.g Chrome, Safari, Mozilla Firefox.)
+- Live Server which is a web extension that helps you to live reload feature for dynamic content (PHP, Node.js, ASPNET)
+
+### Setup
+
+- Using git, clone the code to your machine or download a Zip File.
+
+## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.